manager/integration/tests/test_zone.py

```diff
@@ -0,0 +1,190 @@
+import pytest
+import time
+
+from common import client # NOQA
+from common import core_api  # NOQA
+from common import volume_name # NOQA
+from common import create_and_check_volume
+from common import get_self_host_id
+from common import wait_for_volume_replica_count
+from random import randrange
+from common import wait_for_volume_healthy
+from common import wait_for_volume_degraded
+from test_scheduling import wait_new_replica_ready
+from common import RETRY_COUNTS
+from common import RETRY_INTERVAL
+
+
+ZONE1 = "lh-zone1"
+ZONE2 = "lh-zone2"
+
+K8s_ZONE_LABEL = "topology.kubernetes.io/zone"
+
+
+@pytest.fixture
+def k8s_node_zone_tags(client, core_api):  # NOQA
+
+    lh_nodes = client.list_node()
+
+    node_index = 0
+    for node in lh_nodes:
+        node_name = node.name
+
+        if node_index % 2 == 0:
+            zone = ZONE1
+        else:
+            zone = ZONE2
+
+        payload = {
+            "metadata": {
+                "labels": {
+                    K8s_ZONE_LABEL: zone}
```

Comment:
This label is used for k8s version v1.17 or higher. Need to check the cluster version first then use `failure-domain.beta.kubernetes.io/zone` for the lower version cluster. check [here](https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#topologykubernetesiozone) for details.

Reply:
fixed